code/game/objects/items/RPD.dm

```diff
@@ -14,109 +12,111 @@ RPD
 #define CATEGORY_ATMOS 0
 #define CATEGORY_DISPOSALS 1
 
+
+//find the defines in code\_DEFINES\pipe_construction.dm
+GLOBAL_LIST_INIT(atmos_pipe_recipes, list(
+	"Pipes" = list(
+		new /datum/pipe_info/pipe("Pipe",				/obj/machinery/atmospherics/pipe/simple),
```

Comment:
Do you think it's possible to use initial() fuckery to read name from the path?

Reply:
It totally is, but object names are not the same names that are used in RPD menu.

Reply:
what's more, the order of appearance may not be preserved that way